### DISCUSSION TITLE
/block option configurable

### DISCUSSION THREAD

**Discussion by Jeroen123TC** [Zrips/CMI#7177] [Enhancement]
Now the /block option is not configureable, it is very handy that its possible to have selected blocks who will be blocks with the command /block

a possible configuration format can be:

  craftables:
    - COAL:COAL_BLOCK:9
    - IRON_INGOT:IRON_BLOCK:9
    - REDSTONE:REDSTONE_BLOCK:9
    - GOLD_NUGGET:GOLD_INGOT:9
    - GOLD_INGOT:GOLD_BLOCK:9
    - LAPIS_LAZULI:LAPIS_BLOCK:9
    - EMERALD:EMERALD_BLOCK:9
    - DIAMOND:DIAMOND_BLOCK:9
    - COPPER_INGOT:COPPER_BLOCK:9

**Comment by Zrips** [comment]
/block isint a command of CMI at the moment, could you elaborate more on which command in particular you are talking here?

**Reply by Jeroen123TC** [reply]
I am sorry, we made a alias. The is about the /cmi condense command

**Comment by Zrips** [comment]
Condense is based on existing recipes, if there is one then we will use it to condense or to uncondence items, you do need to have those recipes both ways for it to work. 
What would be the actual benefit in having it predefined in config file? As its already checks for any valid options automatically, manual presets might just introduce issues

**Reply by Jeroen123TC** [reply]
Alright, if its in a config we can control what blocks we want to use ingame with the command. We what that to controll the speed players can level there skyblock islands as example
